client/packages/system/src/Item/api/hooks/useItemVariants/useUnitVariantList.ts

```diff
@@ -4,7 +4,5 @@ import { useItemApi } from '../useItemApi';
 export const useUnitVariantList = () => {
   const api = useItemApi();
 
-  return useQuery(api.keys.unitVariantsList(), () =>
-    api.get.unitVariantsList()
-  );
+  return useQuery(api.keys.packUnits(), () => api.get.packUnits());
```

Comment:
No need to create a new method, the `api.get.packUnits` has no parameters, so you can just:
```suggestion
  return useQuery(api.keys.packUnits(), api.get.packUnits);
```